css/style.css

```diff
@@ -0,0 +1,524 @@
+    border: 1px dotted white;
+    transition-duration: 125ms;
+    transition-property: all;
+}
+    
+.get-started-image:empty {
+    background-image: url('http://i68.tinypic.com/f6lqu.png');
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 380px;
+    height:  380px;
+    border-radius: 100%;   
+}
+
+header {
+    border-bottom: 1px solid black;
+    margin: 35px auto;
+}
+
+.grouping-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 15px auto;
+}
+
+.grouping-container > section {
+    margin: 0 20px;
+}
+
+.middle-img {
+    width: 100%;
+    height: auto;
+    margin: 25px 0;
+}
+
+footer {
+    margin-top: 30px;
+    border-top: 1px solid black;
+    padding-top: 30px;
+}
+
+footer > address {
+    line-height: 1.15;
+}
+
+footer > div.copyright-section {
+    margin: 30px auto;
+    text-align: center;
+}
+
+.header-image {
+    width: 100%;
+    height: auto;
+    margin: 20px 0;
+}
+
+.container > header > .header-description {
+    line-height: 1.5;
+    padding-bottom: 20px;
+    margin: 25px 0 25px;
+}
+
+.container > header > .header-description > h1,
+.container > header > .header-description > h2,
+.container > header > .header-description > h3 {
+    font-size: 37.5px;
+}
+
+.header-description > p {
+    font-size: 20px;
+}
+
+.services {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    width: 100%;
+    border-bottom: 2.5px solid black;
+    margin: 50px 0;
+}
+
+.services > article {
+    width: 35%;
+    background-color: rgb(233, 232, 232);
+    border: 1px solid black;
+    line-height: 1.25;
+    padding: 20px;
+    margin: 37.5px 0;
+}
+
+.services > article > * {
+    margin: 10px 0;
+}
+
+.services > article > h1
+.services > article > h2 {
+    font-size: 20px;
+}
+
+.services > article > a {
+    display: block;
+    border: 1px solid black;
+    border-radius: 5px;
+    text-align: center;
+    width: 100px;
+    height: 40px;
+    margin: 25px 0 15px;
+    background-color: snow;
+    color: black;
+    font-size: 15px;
+    text-decoration: none;
+    padding: 10px;
+}
+
+.extra-information {
+    position: relative;
+    margin: 0 0 75px;
+}
+
+.info-img-right {
+    display: inline-block;
+    width: 40%;
+    position: absolute;
+    right: 0;
+}
+
+.info-img-right > img {
+    width: 420px;
+    height: 300px;
+    position: absolute;
+    right: 0;
+}
+
+.extra-information-has-image {
+    display: inline-block;
+    width: 55%;
+}
+
+.extra-information-has-image > article {
+    width: 100%;
+    line-height: 1.5;
+}
+
+.extra-information-has-image h3 {
+    font-size: 20px;
+}
+
+.extra-information-has-image  ul,
+.extra-information-has-image  ol {
+    line-height: 2;
+    margin: 10px;
+    padding-left: 25px;
+}
+
+.extra-information-has-image ul > li{
+    list-style-type: square;
+}
+
+.extra-information-has-image ol > li{
+    list-style-type: decimal;
+}
+
+@media (max-width: 800px) {
```

Comment:
Really well done media queries... I would only suggest commenting as the solution does.